__tests__/endpoints.test.js

```diff
@@ -0,0 +1,30 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const allData = require("../db/data/test-data");
+
+beforeEach(() => seed(allData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200, an array of topic objects, each of which should have a 'slug' and a 'description' property", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toEqual(expect.any(Array));
+        expect(Object.keys(response.body.topics[0])).toEqual(
```

Comment:
You're checking that the first element in the array conforms to the right object structure here. Nothing wrong with that, but I'd recommend using a .forEach loop here just to make sure that all objects are correct. If you did use a loop though, make sure to check your array's length is more than 0, or the loop will be skipped with a false negative.